9_collaborator_data.yml

```diff
@@ -253,3 +254,28 @@ targets:
       network_ind = '1_network/out/network.rds.ind',
       out_file = target_name)
 
+  #### Reservoirs ####
+
+  res_outflow_ids:
+    command: list(
+      Cannonsville = I('01425000'),
+      Pepacton = I(c('01417000', '01417500')))
```

Comment:
just noting here because I was confused -- one Pepacton site is really close to the reservoir (01417000), and the other is a little downstream (01417500).

Reply:
Thanks. I've added a comment just above this target to clarify for posterity / next month.